src/application/F3DOptionsParser.cxx

```diff
@@ -528,57 +533,65 @@ std::string ConfigurationOptions::GetSystemSettingsDirectory()
 }
 
 //----------------------------------------------------------------------------
-std::string ConfigurationOptions::GetBinarySettingsDirectory()
+std::string ConfigurationOptions::GetBinaryConfigFileDirectory()
 {
-  std::string directoryPath = "";
+  std::filesystem::path dirPath;
   std::string errorMsg, programFilePath;
-  if (vtksys::SystemTools::FindProgramPath(this->Argv[0], programFilePath, errorMsg))
+  try
   {
-    // resolve symlinks
-    programFilePath = vtksys::SystemTools::GetRealPath(programFilePath);
-    directoryPath = vtksys::SystemTools::GetProgramPath(programFilePath);
-    std::string separator;
-#if defined(_WIN32)
-    separator = "\\";
-    if (directoryPath[directoryPath.size() - 1] != separator[0])
-    {
-      directoryPath.append(separator);
-    }
-#else
-    separator = "/";
-    directoryPath += separator;
-#endif
-    directoryPath += "..";
+    // TODO may not work on windows ?
```

Comment:
to check and remove comment if it works

Reply:
works when running from the terminal but does not work when running from the .exe

Reply:
actually works